recipes/cpython/all/test_package/setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+
+# Hack to work around Python 3.8+ secure dll loading
+# See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
```

Comment:
Very tiny change to the test package to see whether the packages are truly relocatable.
```suggestion
# Hack to work around Python 3.8+ secure dll loading:
# see https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
```